roles/installer/tasks/main.yml

```diff
@@ -1,11 +1,10 @@
 ---
-- name: Check for presence of Deployment
+- name: Check for presence of web deployment
   k8s_info:
     api_version: v1
     kind: Deployment
-    name: "{{ ansible_operator_meta.name }}"
+    name: "{{ ansible_operator_meta.name-web }}"
     namespace: "{{ ansible_operator_meta.namespace }}"
-  register: tower_deployment
```

Comment:
this register should still exist or else the following code wont have the `tower_deployment` variable